### DISCUSSION TITLE
sirena per antifurto compatibile con HA

### DISCUSSION THREAD

**Discussion by zuccospooky** [indomus/forum#2362] [Hardware & Firmware]
Ciao a tutti, sto pensando di realizzare un sistema di antifurto seguendo la guida di indomus
https://indomus.it/progetti/realizzare-un-proprio-sistema-dallarme-antifurto-domotico-con-home-assistant/
Vorrei qualche consiglio su un modello di sirena sia da esterno che da interno integrabili in HA, magari con zigbee o z-wave. Quelle indicate dalla guida non sono più disponibili.
Qualcuno si è cimentato in questo progetto? Che sirene avete utilizzato?

Grazie
Fabio

**Comment by M0ntyC0r4** [comment]
Se parliamo di sirena da esterno, la soluzione che ho adottato io, dopo averne vagliate diverse, è:
una qualsiasi sirena che vada con la corrente (se non va a 22o ma va a 12v, un trasformatore da 220 a 12v) ed un sonoff attaccato.
In base alle automazioni la sirena si accende e spenge domoticamente grazie al sonoff.
Tutte le sirene compatibili che avevo trovato online (anche se si parla di qualche anno fa) erano bene o male sirene da interno, da attaccare ad una presa elettrica e soprattutto poco potenti.

Io ho comprato questa ma è piccolina e ad oggi forse non la ricomprerei...ne cercherei una più grane e potente:
https://amzn.to/3KDgtbg

Già questa, secondo me, andrebbe benone: https://amzn.to/3lY4TgE

Per quanto riguarda la sirena da interno, invece, un qualsiasi echo dot, google nest (io uso il nest mini) o altoparlante con jack 3,5 da attaccare direttamente al (-nel mio caso-) raspberry dedicato ad HA e lo si usa per riprodurre il suono della sirena.

**Reply by zuccospooky** [reply]
Perfetto grazie mille

**Reply by MacMattia** [reply]
> Se parliamo di sirena da esterno, la soluzione che ho adottato io, dopo averne vagliate diverse, è: una qualsiasi sirena che vada con la corrente (se non va a 22o ma va a 12v, un trasformatore da 220 a 12v) ed un sonoff attaccato. In base alle automazioni la sirena si accende e spenge domoticamente grazie al sonoff. Tutte le sirene compatibili che avevo trovato online (anche se si parla di qualche anno fa) erano bene o male sirene da interno, da attaccare ad una presa elettrica e soprattutto poco potenti.
> 
> Io ho comprato questa ma è piccolina e ad oggi forse non la ricomprerei...ne cercherei una più grane e potente: https://amzn.to/3KDgtbg
> 
> Già questa, secondo me, andrebbe benone: https://amzn.to/3lY4TgE
> 
> Per quanto riguarda la sirena da interno, invece, un qualsiasi echo dot, google nest (io uso il nest mini) o altoparlante con jack 3,5 da attaccare direttamente al (-nel mio caso-) raspberry dedicato ad HA e lo si usa per riprodurre il suono della sirena.

Ottima soluzione! In che modo si può usare l'uscita audio del Raspberry? Tramite che tipo di entità?

Grazie mille!

**Reply by M0ntyC0r4** [reply]
Eh, allora...intanto ti compri una cassa con Jack da 3,5" e lo colleghi al raspberry. Poi ti scarichi l'addon di vlc e lui ti crea un'entità media_player a cui puoi far riprodurre il suono della sirena che avrai precedentemente scaricato e messo in home assistant (io l'ho messo nella cartella "www").
